nautobot/extras/filters/__init__.py

```diff
@@ -655,13 +655,13 @@ class JobResultFilterSet(BaseFilterSet, CustomFieldModelFilterSetMixin):
         label="Job (ID)",
     )
     obj_type = ContentTypeFilter()
-    created = django_filters.DateTimeFilter()
-    completed = django_filters.DateTimeFilter()
+    date_created = django_filters.DateTimeFilter()
+    date_done = django_filters.DateTimeFilter()
     status = django_filters.MultipleChoiceFilter(choices=JobResultStatusChoices, null_value=None)
 
     class Meta:
         model = JobResult
-        fields = ["id", "created", "completed", "status", "user", "obj_type", "name"]
+        fields = ["id", "date_created", "date_done", "name", "status", "user", "obj_type"]
```

Comment:
Let's be sure to add either as a requirement to this PR or a follow-on to create the migration docs for all the model changes here.

Probably would be a good pre-release checklist item is to review every migration in `next` and ensure we have a migration guide time around it.